content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -5,69 +5,77 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-To make sure that Redis Enterprise Software (RS) servers can pass necessary communications between them,
-we recommend that all RS servers have all of the ports listed here open
-between them.
+To make sure Redis Enterprise Software servers can pass necessary communications between them,
+we recommend all servers have the following ports open.
 
-By default, the cluster assigns ports in the range of 10,000 - 19,999
-to database endpoints. If you assign a specific port for a database when
-you create it, even outside of this range, the cluster only verifies
-that the assigned port is not already in use. You must manually
-update your firewall with the port for that new database endpoint.
+{{% Note %}}
+You must manually update your firewall with the port for each new database endpoint. The cluster assigns ports (between 10,000 and 19,999) for database endpoints by default, and only verifies that new endpoints are not already in use.
+{{% /Note %}}
 
 ## Ports and port ranges used by Redis Enterprise Software
 
+There are 3 types of ports that are open for use by RS:
+
+- Internal - The traffic is between or within the cluster nodes
+- External - The traffic is from client applications or external monitoring resources
+- Active-Active - The traffic is from clusters that host Active-Active databases
+
+### Ports for traffic between the cluster and other systems
```

Comment:
@kaitlynmichael I find it really confusing that we describe Internal, External, and Active-Active ports but then don't stick with that organization scheme as we describe the ports below.

Reply:
Agreed. I didn't consider that in my first pass.

Reply:
I combined the table into one since the internal/external/active-active distinction is already made within the table. We don't need the extra headings